File: divsum/python3/divsum.py
import math

t = input()
for t in range(int(t)):
    num = int(input())
    divsum = 0

    i = 2
    while i <= math.sqrt(num):
        if(num % i == 0):
            if(i == (num / i)):
                divsum += i
            else:
                divsum += int(i + num / i)
        i += 1

    if(num > 1):
        divsum += 1
        
    print(divsum)

File: divsum/golang/divsum.go
package main

import (
	"fmt"
	"math"
)

func main() {
	var testCase int
	fmt.Scanf("%d", &testCase)
	for t := 0; t < testCase; t++ {
		var num int
		fmt.Scanf("%d", &num)
		divsum := 0

		limit := int(math.Sqrt(float64(num)))
		for i := 2; i <= limit; i++ {
			if num%i == 0 {
				if i == (num / i) {
					divsum += i
				} else {
					divsum += (i + num/i)
				}
			}
		}
		if num > 1 {
			divsum++
		}
		fmt.Println(divsum)
	}
}
